### DISCUSSION TITLE
Why is `null` a possible value when using `VariantProps`?

### DISCUSSION THREAD

**Discussion by sunwrobert** [joe-bell/cva#223] [Feedback]
I feel like the value should either be there or `undefined`. I currently have my own custom `VariantProps` around this but feel like this should be built into this library IMO. Thoughts?

**Comment by ColinKrist** [comment]
https://beta.cva.style/getting-started/whats-new
![image](https://github.com/joe-bell/cva/assets/27971797/65c674a0-8973-410c-909f-f7b55448bae6)
